### DISCUSSION TITLE
What to do with hammerjs

### DISCUSSION THREAD

**Discussion by natew** [software-mansion/react-native-gesture-handler#2150] [Ideas]
On the web, hammerjs is not terrible, but definitely has some cost. It's also unmaintained, [according to this](https://github.com/hammerjs/hammer.js/issues/1278).

Seems like someone has started on [contact.js](https://github.com/biodiv/contactjs) as a replacement. It seems fairly nice and they've been active.

It'd be good to try and guide the development of the web stuff towards as much tree shaking as possible, and further reducing bundle size. Happy to help.

cc @SE2Dev @biodiv

**Comment by m-bert** [comment]
Hi! We took care of it and in the [2.6.0](https://github.com/software-mansion/react-native-gesture-handler/releases/tag/2.6.0) version we have introduced new web implementation. Current version [2.6.1](https://github.com/software-mansion/react-native-gesture-handler/releases/tag/2.6.1) contains many fixes. I suggest trying it out by calling `enableExperimentalWebImplementation()` in the root of the project.

**Reply by natew** [reply]
Very cool 😎
